Culture Convos 1 & 2 from TheMarksman (#6493)
endless-sky/endless-sky: data/human/culture conversations.txt

mission "Worker's Occasion"
	minor
	source
		government "Republic"
		attributes "dirt belt"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation

> 			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at its menu, hand shaking as he walked in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`

Reviewer: > Pausing to consider a restaurant for your dinner, you notice a local worker staring at its menu, ...

At first glance i thought the "its" was referring to the worker. "the menu" or a more elaborate version would work just as well.

> hand shaking as he walked in.

Bit odd since you just noticed him staring, yet you're supposed to have seen him walk in?

Author: The pitfalls of hastily proofreading a draft. XD
```suggestion
			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`
```

> 			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`

Reviewer 2: ```suggestion
			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you, but you recall how unreachable they would've been for you before your career change.`
```

> 			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`

Reviewer 2: Does neatly worn here refer to wear and tear or how the man is wearing his clothes?

Author: How he's wearing them, the freshly brushed is their condition.

Reviewer 2: I think I was just overanalyzing here, this doesn't need to be changed.

> 			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you, but you recall how unreachable they would've been for you before your career change.`

Reviewer 3: ```suggestion
			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn and look freshly brushed, but it's impossible not to notice the dirty marks and stains that haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize that while the prices are very affordable for you, they would've been unreachable before your career change.`
```

Reviewer 4: The description feels a little odd to me here. The vibe I get is that this guy's hiding something and doing it poorly, not that he's just paying a tall price for a rare fancy dinner.
```suggestion
			`Pausing to consider a restaurant for your dinner, you notice a local worker with a small handful of credits eagerly eyeing the menu. His clothes are neatly worn and look freshly brushed, but it's impossible not to notice the dirty marks and stains that haven't washed out. The man undeniably seems nervous, and he looks at the formal dining layout with an obvious lack of familiarity. Looking at the menu yourself, you realize that while the prices are very affordable for you, they would've been unreachable before your career change.`
```
Thoughts?

Reviewer 5: Seems good.

			choice

> 				`	(Offer to buy the man's dinner.)`

Reviewer 2: ```suggestion
				`	(Offer to pay for the man's dinner.)`
```
Sounds like you're buying it from him, not buying it for him

					goto aid

> 				`	(Pity the man.)`

Reviewer: Might just be me, but on first read i didn't recognize this as the passive option.

Reviewer 2: I had the same confusion.

Reviewer: ```suggestion
				`	(Silently pity the man.)`
```

					goto pity
				`	(Smile at the man.)`

> 			`	You smile at the man and reflect on the value of small achievements. He catches your smile out of the corner of his eye and gives you a hesitant but friendly wave back before you go back about your business.`

Reviewer 2: I feel like should be "go back to your business" or "go about your business", not a combination of both.

Author: ```suggestion
			`	You smile at the man and reflect on the value of small achievements. He catches your smile out of the corner of his eye and gives you a hesitant but friendly wave back before you go about your business.`
```

				decline
			label aid

> 			`	You enter the restaurant and catch the man's attention, offering to buy his meal.`

Reviewer 2: ```suggestion
			`	You enter the restaurant and catch the man's attention, offering to pay for his meal.`
```

> 			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. On reflection, you think that perhaps your pride would've been rather heavily wounded were you in his position.`

Reviewer 3: Not a big fan of telling the player what they should be feeling.
```suggestion
			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. As you walk away, you think about what your reaction would be if a much wealthier individual pitied you by paying for your dinner.`
```

Author: Mmm, maybe, but it is only "perhaps", and at this stage the player chose the dumbass option

Reviewer 4: Lol I would've unironically chosen the dumbass option trying to be nice
I do agree that this is probably good as-is though.

Reviewer 3: Maybe it's my own personal disposition, but that line really took me out of the conversation, especially when considering how the rest of the game is generally written.

> 			`	He looks at your pilot's attire and frowns immediately before telling you where you can stick that offer in colorful terms. On reflection you think that perhaps your pride would've been rather heavily wounded were you in his position too.`

Reviewer 2: ```suggestion
			`	He looks at your pilot's attire and frowns immediately before telling you where you can stick that offer in colorful terms. On reflection, you think that perhaps your pride would've been rather heavily wounded were you in his position.`
```
The too at the end makes it seem a little wordy.

> 			`	He looks at your pilot's attire and frowns immediately before telling you where you can stick that offer in colorful terms. On reflection you think that perhaps your pride would've been rather heavily wounded were you in his position too.`

Reviewer 2: "your pilot's attire" makes it sound like he's talking about the attire of your pilot to me.

Author: hmmmmmm

Reviewer: ```suggestion
			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. On reflection you think that perhaps your pride would've been rather heavily wounded were you in his position too.`
```

```suggestion
			`	He looks looks you over for a second, before frowning and telling you where you can stick that offer in colorful terms. On reflection you think that perhaps your pride would've been rather heavily wounded were you in his position too.`
```

Scowl would be another choice, though it doesn't help with the problem at hand.

Author: ```suggestion
			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. On reflection, you think that perhaps your pride would've been rather heavily wounded were you in his position.`
```

				decline
			label pity

> 			`	You pity the man who as to work a full month to afford this and continue on your business.`

Reviewer: ```suggestion
			`	You pity the man who has to work a full month to afford this and continue on your business.`
```

> 			`	You pity the man who has to work a full month to afford this and continue on your business.`

Reviewer 2: ```suggestion
			`	You pity the man who has to work a full month to afford this and continue on your way.`
```
"Continue on your business sounds off to me, this feels more natural.

mission "Forest Fire"
	minor
	source
		government "Free Worlds"
		attributes "frontier"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation

> 			`The smell of burning smoke permeates the air as you step off your craft, causing you to rush towards the nearest shelter, coughing along the way. Once there, you find people shouting, rushing about to get tape plastered onto the windows. No one seems to have noticed your arrival and none give an explanation.`

Reviewer 2: ```suggestion
			`The smell of burning smoke permeates the air as you step off your craft, causing you to rush towards the nearest shelter, coughing along the way. Once there, you find people shouting and rushing about to get tape plastered onto the windows. No one seems to have noticed your arrival and none give an explanation.`
```

			choice

> 				`	"Hey! What's happening? I can help!" (Shout.)`

Reviewer 3: ```suggestion
				`	"Hey! What's happening? I can help!"`
```

					goto help
				`	(Look for someone not in motion.)`

> 			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`

Reviewer 2: "blow over us" doesn't seem like a natural way to describe a fire.

Author: This might be an Australian phrase. If anyone has a more broadly used alternative I'll accept that.

Reviewer 2: Hoping something "blows over" as in hoping it stops being an issue is definitely a phrase people use in the US. The issue is that, especially because you mentioned the wind previously, it sounds like they're hoping the wind blows the fire over them, which isn't really how fires work. If that was the intent, this might work?
```suggestion
			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping this blow over."`
```

Author: But... it is how fire works? Like if you hunker down in a secure (non-flammable) shelter in the face of a firefront being driven by wind, the front literally blows over you.

Reviewer 2: I guess my thought was that fire doesn't really skip an area. Maybe it would burn around the shelter without harming it, as it is fire-proof, but it doesn't seem like it would just blow over an area without touching it.
However, this is purely based on intuition and is based on no actual knowledge of fire. Feel free to ignore this suggestion.

Author: > but it doesn't seem like it would just blow over an area without touching it.

Yeah you'd be surprised, I think you'd be fascinated by what does and doesn't get burnt in big wind-driven fires.

Reviewer 4: ```suggestion
			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll burn out before it does too much damage."`
```
How about this? I do generally agree with Arrow on "blowing over us" being an odd way to describe this fire.

Reviewer 5: Let's go politically neutral and just take the change.

> 			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or one some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`

Reviewer: ```suggestion
			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`
```

> 			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or one some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`

Reviewer: > You seek out a pregnant lady sitting on a chair, a mask worn as she explains:

The "a mask worn" caught me off guard. is that valid?

Reviewer 5: It may or may not be valid, depending on how you interpreted it.
Worst case scenario, just says "You seek out a masked pregnant lady" or "You seek out a masked lady sitting on a chair, hands placed protectively over her belly" or some other ways to portray the idea.

			choice

> 				`	"I have a ship out there. Let me help."`

Author: It just avoids having the game put words in your mouth without the player doing anything about it.

> 				`	"I have a ship out there. Let me help."`

Reviewer 6: Seeing as this isn't meant to be a long mission chain, I get why there wouldn't be branching dialog here. Feel free to resolve

			`	Her eyes widen and she looks around, sharply catching the attention of one of the many people hurrying about.`
				goto fight
			label help
			`	A young man who looks suspiciously like he might've been air traffic control twenty minutes ago glances at you and skids to a stop. "Oh, you're a pilot! Can your ship help?"`
			choice

> 				`	"Yes."`

Reviewer 6: Same issue as above.

			label fight

> 			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the Air Traffic Controller clears you to depart, navigating your craft towards the fire.`

Reviewer 6: ```suggestion
			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the air traffic controller clears you to depart. You navigate your craft towards the fire.`
```

Reviewer 6: Can you elaborate on why you don't like the suggested change?

Reviewer 5: Air Traffic Controller is a title and hence capitalised. It's also not good practice to cut the sentence short and add yet another short sentence at the back when you can join them together.

Reviewer 6: Would this be better?
```suggestion
			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the Air Traffic Controller clears you to depart, and you navigate your craft towards the fire.`
```

Author: To me it now it feels clumsily constructed.

Reviewer 6: You can resolve this then if you like.

> 			`	The ragging fire flared like an inferno, throwing up black smoke as you approached. Animals scrambled towards safer grounds as you dropped the retardants, making sure to keep your speed low before going back for another batch.`

Reviewer: Seems odd to switch to past tense all of a sudden. The time skip can either happen between paragraphs, or you can end this sequence with a dialog option a là "keep dropping retardant" to make it more noticeable.

Author: Proposal:
```suggestion
			`	The dock-hands scramble to get the firefighting equipment onto your ship after your proclamation, instructing you to drop the retardants onto the perimeter of the fire. You barely manage to remember what every button does before the Air Traffic Controller clears you to depart, navigating your craft towards the fire.`
			`	The raging fire flares like an inferno, throwing up black smoke in deep billowing waves as you approach. As you drop the retardants, making sure to keep your speed low, animals scramble towards safer grounds. You try to make sure you've not emptied right on top of them each time as you go back for another batch.`
```

Reviewer: lgtm

> 			`	Hours later, you look at your handiwork, a long line of red keeping back the flames as the fire starts to smother out. You radio your progress and receive a cherry response, thanking you for your help.`

Reviewer: cherry response? never heard that before.

Author: ```suggestion
			`	Hours later, you look at your handiwork, a long line of red keeping back the flames as the fire starts to smother out. You radio your progress and receive a cheery response, thanking you for your help.`
```

Reviewer: Oh. 😄
